rust/agents/scraper/src/scraper/mod.rs

```diff
@@ -795,39 +824,49 @@ impl SqlChainScraper {
             let _ = blocks
                 .get_mut(&hash)
                 .expect("We found a block that we did not request")
-                .insert((Some(block.id), block.timestamp));
+                .insert((
+                    Some(block.id),
+                    BlockInfo {
+                        hash,
+                        timestamp: date_time::to_unix_timestamp_s(&block.timestamp),
+                        // TODO: we don't actually use these below, we should make sure to clean
+                        // this out
+                        number: 0,
```

Comment:
why can't we set this? or is this something that's resolved by the refactor? or can we just rm for now

Reply:
Yes, this gets resolved.